src/components/ng-redux.ts

```diff
@@ -52,11 +58,20 @@ export class NgRedux<RootState> implements ObservableStore<RootState> {
    * @param enhancers Optional Redux store enhancers
    */
   configureStore(
-    rootReducer: Reducer<RootState>,
-    initState: RootState,
+    rootReducer?: Reducer<RootState>,
+    initState?: RootState,
```

Comment:
`rootReducer` is now optional because there is an alternate way of providing it through `forRoot`. `initState` has always been optional in my mind since your reducers compute your initial state without a seed anyways. By making these optional, I was able to implement the feature without breaking the API